extensions/util.py

```diff
@@ -0,0 +1,20 @@
+import discord
```

Comment:
Maybe we can create a sub dir for utils, because I have planned such a structure for the rework as well. So we can differentiate the utils for different extensions

Reply:
Isn't the point of a util package to include **common** utility functions? If they are only useful for a single extension, then there would be no need for a separate file. I might be missing the point of your suggestion :D

Reply:
At this point I only those functions useful for the tournament function. But we can keep it like this and I but my helper functions in here too. My thought in the first place was that we might have overhead because of imports for functions we do not need e.g when an extension is not even loaded

Reply:
I agree with @KnutZuidema there. It makes more sense to have a utility file for including common utility functions.